python/lsst/pipe/tasks/fit_coadd_multiband.py

```diff
@@ -198,9 +212,19 @@ def adjustQuantum(self, inputs, outputs, label, data_id):
         return adjusted_inputs, {}
 
     def __init__(self, *, config=None):
+        super().__init__(config=config)
+        if config is None:
+            return
```

Comment:
FWIW I've gotten the habit just doing
```
assert isinstance(config, DerivedConfigClass)`
```
here; I think the `=None` in the base class is just unnecessary, but I haven't ever gotten around to fixing it.